### PR TITLE
Update runastrodriz to work under Windows

### PR DESCRIPTION
**Pull request by stsci-hack**
These fairly simple, and blunt, changes address problems that prevented `runastrodriz` from successfully executing to completion under Windows (the OS).  The changes are:
* insure that `num_cores` parameter value as set by the user gets passed to `astrodrizzle` when generating the a posteriori drizzle products 
* added `try...except` block around the deletion of the astrodrizzle log file in `run_driz`
* A note was added to the `runastrodriz` sphinx docs to suggest that users set `num_cores` to 1 when running under Windows due to problems with threaded logging (or, in fact, lack of any working logging when using multiprocessing as we have it implemented now in `astrodrizzle`). 

In addition, a few (PEP8) Flake8 syntax errors were also corrected.

### CODE REVIEW

File: drizzlepac/runastrodriz.py [hunk 1]
```diff
@@ -1065,19 +1066,19 @@ def verify_alignment(inlist, calfiles, calfiles_flc, trlfile,
...
 def apply_headerlet(filename, headerlet_file, flcfile=None):
-    
-    # Use headerlet module to apply headerlet as PRIMARY WCS 
+
+    # Use headerlet module to apply headerlet as PRIMARY WCS
     headerlet.apply_headerlet_as_primary(filename, headerlet_file,
                                         attach=True, archive=True)
     # Verify that all keywords from headerlet got applied
-    hlet = headerlet.Headerlet.fromfile(headerlet_file)
+    # hlet = headerlet.Headerlet.fromfile(headerlet_file)
```
Comment (mdlpstsci) [inline drizzlepac/runastrodriz.py:1074]: Is this meant to be commented out?

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
One question which needs to be answered and otherwise this PR is ready for merging.

File: drizzlepac/runastrodriz.py (hunk 1 above)
Comment (stsci-hack) [reply drizzlepac/runastrodriz.py]: Fixed under 11fdea6.

**Comment by stsci-hack** [comment]
The changes in this PR now enforce single-CPU operations when running under Windows for both AstroDrizzle (single-drizzle step) and runsinglehap for the upcoming SVM processing.  This will need to be revised once the logging can be updated by resolving #799.

**Pull request merged**